homeassistant/components/overkiz/button.py

```diff
@@ -27,20 +27,20 @@ class OverkizButtonDescription(ButtonEntityDescription):
     # My Position (cover, light)
     OverkizButtonDescription(
         key="my",
-        name="My Position",
+        name="My position",
         icon="mdi:star",
     ),
     # Identify
     OverkizButtonDescription(
         key="identify",  # startIdentify and identify are reversed... Swap this when fixed in API.
-        name="Start Identify",
+        name="Start identify",
```

Comment:
Why not name this Identify? That's the action performed when the button is pressed

Reply:
Hmn I see now there is also a Stop button. Why not make that a toggle ? Or just automatically turn it off after X time?

Reply:
Their is also an Identify method that does exactly what you want: start, wait a little and stop. 
I have open an issue to Somfy about their weird naming, but apparently my request is not clear: https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/30

Reply:
do we need to expose it all or can we just expose the identify command that actually runs identify for a couple of seconds?

Reply:
We disable most of them by default; but I believe we added the extra Start / Stop for users that would have this functionality for a longer time. For example, to have a light that keeps blinking until they stop it.

In theory, these two buttons could be moved to a toggle switch. Didn't think about that yet! :-)

Reply:
Make it a switch and disabled by default 👍